README.md

```diff
@@ -203,7 +203,7 @@ branch`.
 Version numbers should be bumped with `changeset` and git tags should be pushed
 to the release branch BEFORE merging.
 
-1. Run `pnpm changeset version` from root to bump versions
+1. Run `pnpm version:changeset <adaptor-name>` from root to bump versions and add release dates
```

Comment:
hmm, one obvious problem here is that `pnpm changeset version` will run for all adaptors that need their versions updating. If you bump common, that means it'll update version numbers for every adaptor. 

But this common only updates dates in the changelog for a single adaptor